pyproject.toml

```diff
@@ -0,0 +1,31 @@
+[tool.poetry]
+name = "perf_tools"
+version = "0.1.0"
+description = ""
+authors = ["Sébastien Eustace <sebastien@eustace.io>"]
+
+[tool.poetry.dependencies]
+python = "^3.7"
+plac = "^1.1"
+geojson = "^2.5.0"
+shapely = "^1.7.0"
+folium = "^0.10.1"
+scikit-learn = "^0.22.2"
+mlrose = "^1.3.0"
+jsonpickle = "^1.3"
+aiohttp-index = "^0.1"
+matplotlib = "^3.2.1"
+colorhash = "^1.0.2"
+jsonpath-ng = "^1.5.1"
+pandas = "^1.1.2"
+dask = "^2.26.0"
+numpy = "^1.19.2"
+toolz = "^0.10.0"
+tornado = "^6.0.4"
```

Comment:
don't think we need tornado / fsspec?

Reply:
Installing dask using `poetry add dask` not all modules are installed, as mentioned here https://docs.dask.org/en/latest/install.html#pip . Tests fail with `ImportError: Dask dataframe requirements are not installed.` until `fsspec` and`fsspec` are installed

Reply:
Installing Dask using `poetry add dask` misses some modules https://docs.dask.org/en/latest/install.html#pip. Tests then fail with `ImportError: Dask dataframe requirements are not installed.` until `toolz` and `fsspec` are installed